cs-minibot-platform-src/src/main/java/examples/gobot/Course.java

```diff
@@ -0,0 +1,83 @@
+package examples.gobot;
+
+import basestation.vision.VisionCoordinate;
+
+import java.awt.geom.Path2D;
+import java.util.ArrayList;
+
+public class Course {
+    private Border inner;
+    private Border outer;
+    private Path2D.Double startArea;
+    //private Path2D.Double finishArea;
+    private Path2D.Double middleArea; //sketchy anti-cheating mechanism
+    private static final double MARGIN = 0.2;
+    private static final double TIMEALLOWEDOUT = 1.0;
+    private double timeOutside = 0;
+
+    public Course() {
```

Comment:
Forgot, but I think you can just not have a constructor if it would be empty anyway.